Project.toml

```diff
@@ -51,7 +51,7 @@ NaNMath = "0.3, 1"
 PGFPlotsX = "1"
 PlotThemes = "2, 3"
 PlotUtils = "1"
-PlotlyBase = "0.7"
+PlotlyBase = "0.7 - 0.8"
```

Comment:
@BeastyBlacksmith, is that safe ?

Xref https://github.com/sglyon/PlotlyBase.jl/pull/100.